pgjdbc/src/test/java/org/postgresql/test/jdbc2/ServerErrorTest.java

```diff
@@ -31,6 +33,7 @@
   @Before
   public void setUp() throws Exception {
     con = TestUtil.openDB();
+    assumeTrue(TestUtil.haveMinimumServerVersion(con, ServerVersion.v9_3));
```

Comment:
As an idea: might be introduce a method like `assumeMinimumServerVersion(con, ServerVersion.v9_3)` ?